### DISCUSSION TITLE
Next features

### DISCUSSION THREAD

**Discussion by CarlosLugones** [lugodev/ptb-django-cookiecutter#9] [General]
Let's talk about the next features to be added to the cookiecutter.

**Comment by CarlosLugones** [comment]
@ragnarok22 qué cambios pensabas hacer?

**Reply by ragnarok22** [reply]
creo que el `engine.py` no debería ser tocado por el usuario. Se podría implementar aparte donde se declaran los commands, conversations, etc y el `engine.py` que los llame

**Comment by ragnarok22** [comment]
Great! This could be the next steps:
- [ ]  internationalization
- [ ] continous deployment
- [ ] continous integrations
- [ ] make configuration in other side, not in engine.py
